### PR TITLE
SecurityPkg to CI and IntrinsicLib from CryptoPkg to MdePkg

### PR DESCRIPTION
**Pull request by MikhailKrichanov**

### CODE REVIEW

File: .gitmodules [hunk 1]
```diff
@@ -23,9 +23,6 @@
 [submodule "UnitTestFrameworkPkg/Library/GoogleTestLib/googletest"]
 	path = UnitTestFrameworkPkg/Library/GoogleTestLib/googletest
 	url = https://github.com/google/googletest.git
-[submodule "UnitTestFrameworkPkg/Library/SubhookLib/subhook"]
-	path = UnitTestFrameworkPkg/Library/SubhookLib/subhook
-	url = https://github.com/Zeex/subhook.git
```
Comment (mikebeaton) [inline .gitmodules:28]: I think the subhook fix should be a separate PR?

Comment (mikebeaton) [reply .gitmodules:28]: (ref https://github.com/acidanthera/bugtracker/issues/2441)

Comment (MikhailKrichanov) [reply .gitmodules:28]: There is no use in a broken link (https://github.com/Zeex/subhook), is it not?

Comment (mikebeaton) [reply .gitmodules:28]: No, there is no use in a broken link, but I suggested:

1. This is completely separate from IntrinsicLib changes and should be a separate PR.
2. We do not know how they will resolve it upstream - for instance they might dig out a checked-out version and host is themselves. So rather than manually dig in and remove a bunch of files, we could just drop the whole parent submodule quietly for now, in our own build chain.

Comment (mikebeaton) [reply .gitmodules:28]: Anyway, it's only a suggestion, and if you prefer to remove it completely at this stage, I guess it's fine. (I am not entirely sure how useful the parent submodule is without this - but at least this fixes our build.)
